pkg/log/buffalo_logger.go

```diff
@@ -12,9 +12,13 @@ import (
 // It's also good to note that internal logs
 // from buffalo should only be allowed in development
 // as our logging-system should be handled from our codebase.
-func Buffalo() buffalo.Logger {
+// If the environment is production, then its a noop logger.
+func Buffalo(env string) buffalo.Logger {
 	l := logrus.New()
 	l.Formatter = &buffaloFormatter{}
+	if env == "production" {
```

Comment:
Is it not possible to drain the production logs somewhere?
Cloud monitoring systems won't be able to diagnose cases where panics happen only in production but not in other modes?

Reply:
Just to be clear, we might not want to have drain somewhere in MVP

Reply:
@manugupt1 this is just buffalo logs, not application logs. Buffalo spits out internal logs such as every single request that comes in. It clutters the cloud monitoring systems. 
However, you're right about panics. We should capture those. Unfortunately buffalo swallows panics and tries to log them with missing info, but we should at least log whatever it gives us. So I've updated this pr. Thanks!